tests/test_amqp.py

```diff
@@ -1071,7 +1071,7 @@ def share(*a, **kw):
 
         await queue.bind(exchange, routing_key)
 
-        body = bytes(shortuuid.uuid(), 'utf-8') * 9999999
+        body = bytes(shortuuid.uuid(), 'utf-8') * 6000000
```

Comment:
Why it’s affected?

Reply:
I've mentioned it in commit body: `Since RabbitMQ 3.8.0 max size of message is 128MB.` Otherwise test fails.